### ISSUE TITLE
Changes to intro/exit screens

### ISSUE THREAD

**Issue by noah10** [EmotionCognitionLab/pvs#38]
Some tasks have intro screens that aren't very informative ("Welcome to PANAS") and others have exit screens that provide a pause but also are just another thing to click through. We should standardize these so that:

1. Intro screens always contain instructions. In some cases (i.e. questionnaires) the instructions are so minimal that the "intro screen" will just be the task itself.
2. Intro screens should say "Thank you for completing the previous task" at the top, so that we combine the exit for one with the intro to the next.

**Comment by noah10** [comment]
This has gotten a little trickier. We can't say "Thank you for completing the previous task" on the first task. Back when the health assessment was always the first task that wasn't a problem. Now that that's not the case, though, we have a different first task every day. In order to save the time and trouble of coding things so that every task knows whether it's the first one of the day or not (which, if we're finicky, could also depend on whether the user finished all of the tasks the previous day) we should probably either (a) come up with different language that works regardless of whether the task is first or not or (b) just drop requirement 2 above.

@maramather suggested "Welcome to the next task", which is an improvement (although maybe still slightly awkward on the first one). @nashiro or @ycazaherrera , do either of you have other suggestions? Or do we think we should just skip requirement 2 altogether?

**Comment by ycazaherrera** [comment]
Could we add one screen to the beginning of each task that says something like "Press the spacebar to start a new task"? After pressing the spacebar participants would see the original intro screen and instructions for the task they are about to begin.

**Comment by noah10** [comment]
We mostly already have that - most of the tasks have exit screens that say something like "press the spacebar to continue". This issue was actually opened in an effort to cut back on the number of intro/exit screens we have. I suppose we could remove intro screens (where possible) and keep the exit screens, though that is more stuff for participants to wade through.

**Comment by ycazaherrera** [comment]
Right. How about saying "You are about to start a new task" in place of "Thank you for completing the previous task"?

**Comment by noah10** [comment]
I like that better than "Welcome to the next task." Unless anyone has any other ideas (or problems with that one) I'll go with that.

**Comment by nashiro** [comment]
"You are about to start a new task" sounds good to me too.

**Comment by noah10** [comment]
This should now be done. Testing it will require doing at least 5 days of all of the daily tasks, because (a) this has to be evaluated to make sure it makes sense when you're going from one task to another and (b) because verbal-learning doesn't turn up until set five and (c) many of the tasks have different intro screens the first and second times they appear. Is there anyone available who could do all the daily tasks each day for at least 5 days, @nashiro ?

**Comment by nashiro** [comment]
I'm thinking asking Paul (choipaul@usc.edu) if he would be willing to do all the daily tasks each day for at least 5 days. Can I link him to this thread?
@choipaul@usc.edu

**Comment by choipaul92** [comment]
@noah10  Just to clarify, do I have to do all of the individual tasks + "all tasks" (https://dev.heartbeamstudy.org/index.html)

**Comment by noah10** [comment]
@choipaul92 No - just go to https://dev.heartbeamstudy.org/daily-tasks/ each day and do tasks until it tells you you're done for the day. It would be best if you didn't start this until Tuesday morning, as I'm going to re-arrange the tasks to match (more or less) what's in the spreadsheet then, and if you start before then you'll have to start over.

**Comment by choipaul92** [comment]
@noah10, okay. I'll check back on Tuesday morning using that link!

**Comment by nashiro** [comment]
Thanks so much Paul!

**Comment by ycazaherrera** [comment]
@noah10 My RA was trying to do the daily tasks this morning and was not able to run them on Chrome. She tried doing shift+reload but that still did not work. 

Note: I am adding this comment here since there was an existing discussion on the topic. Let me know if you want me to create a new issue for this.

**Comment by choipaul92** [comment]
@noah10, I noticed all of the tasks were white background and black text (except for task-switching); did we decide on the beige background?

**Comment by noah10** [comment]
> @noah10, I noticed all of the tasks were white background and black text (except for task-switching); did we decide on the beige background?

See #43 .

**Comment by choipaul92** [comment]
@noah10, I cannot access the daily tasks.

**Comment by choipaul92** [comment]
I noticed that there were issues in the background during questionnaires (PANAS + Daily Stressor + FFMQ) where the beige background would cut off after a certain question and the rest would be white. For the Eyes Test, I notice that there was a black box on the lower right-hand corner of the image giving a definition of the word located in the lower right hand corner along with an example of a sentence. After I finished the Face Name Recognition Task, I received "The experiment failed to load" but that could be due to because I lost my wifi. I refreshed it and led me back to "pattern separation task" which I did before.
